### ISSUE TITLE
SteemD: Create a consistent pattern for testnet related api values

### ISSUE THREAD

**Issue by relativityboy** [steemit/steem#2882]
As a maintainer of Steem, I want Mainnet to be as secure as possible *so that everyone invested in Steem is "safe"*. I also want ease of development, testing and deployment with a minimum of spurious outcomes in those activities, *so I can be efficient and product higher quality results* 

As a client connecting to a testnet (node, lib, user doing curl requests), I want to have a way of detecting clearly, if I'm running on a testnet or not, and a way to know what important values have been set at compile time, and what values have been set at boot time (eg, cl-arguments or config file params)

1. An api endpoint that is only available when testnet is running ex: `testnet_config` that when testnet divulges testnet configuration info. Returns error otherwise.
1. `database_api.get_config` returns 'compile time' values in addition to 'boot time' values. This allows clients to do a parity check in a single call if they're connecting to an untrusted node.

**AC**
- [ ] in `# Results`  evaluate the above options, and decide if either design is better than the current solution

**Comment by mvandeberg** [comment]
> `database_api.get_config` returns 'compile time' values in addition to 'boot time' values. This allows clients to do a parity check in a single call if they're connecting to an untrusted node.

This is not true. `get_config` **only** returns compile time values. It literally just returns https://github.com/steemit/steem/blob/master/libraries/protocol/include/steem/protocol/config.hpp as json.

**Comment by relativityboy** [comment]
In the **AC** 

> evaluate the above options ....

**Comment by mvandeberg** [comment]
In `get_config`, the value for `STEEM_ADDRESS_PREFIX` is `TST` on testnet and `STM` on mainnet. So there are already ways, if you trust the node, to programmatically determine if you are connected to a testnet node or not.
